src/harness/unittests/cachingInServerLSHost.ts

```diff
@@ -123,7 +123,7 @@ namespace ts {
                 try {
                     // trigger synchronization to make sure that LSHost will try to find 'f2' module on disk
                     project.getLanguageService().getSemanticDiagnostics(imported.name);
-                    assert.isTrue(false, `should not find file '${imported.name}'`);
+                    assert.fail(false, `should not find file '${imported.name}'`);

```

Comment:
`assert.fail` takes `actual` and `expected` as parameters. Doesn't seem like there's an overload that just takes a message. `assert(false, msg)` is probably best.